### ISSUE TITLE
Support of ZRANGE command

### ISSUE THREAD

**Issue by alex6058** [RedisGrafana/grafana-redis-datasource#182]
Introduce new command `ZRANGE` in the `Redis` commands category

Command parameters: 
`key`
`selectby` (options: `BYINDEX`|`BYSCORE`|`BYLEX` `BYINDEX` is default)
`reverse order` checkbox 
min (default values: `0` for `BYINDEX`, `-inf` for `BYSCORE`, `-` for `BYLEX`)
max (default values: `-1` for `BYINDEX`, `+inf` for `BYSCORE`, `+` for `BYLEX`)
`WITHSCORES` mandatory on

The expected output of the command with `WITHSCORES` flag is

```
>> zrange test:zset 0 -1 withscores

1) "member1"
2) "10"
3) "member2"
4) "12"
5) "member3"
6) "14"
7) "member4"
8) "15"
``` 

This output to be parsed same way as `HGETALL` output to be able to be used in Bar gauge, Stat, Table and other compatible Grafana panels

**Comment by DeoLeung** [comment]
is there a way that I can manually format the universal input result as table before it's predefined?

**Comment by mikhailredis** [comment]
@alex6058 @DeoLeung ZRANGE implemented and will be released in version 2.0 this week.
